src/index.html

```diff
@@ -4,14 +4,216 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action=
+      "https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <!-- Personal information -->
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
```

Comment:
do we need this comment?

Reply:
No, its only for me. Well i can remove these comments.

Reply:
For the next tasks, can I write commets, or I can't do it?

Reply:
> No, it's only for me.

In tasks, you write code like for production, and do we need these comments in prod?

Reply:
> For the next tasks, can I write commets, or I can't do it?

Yep, but where it is really needed

Reply:
How can i learn about commets for production?